sos/report/plugins/curtin.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Curtin(Plugin, IndependentPlugin):
+
+    short_desc = 'GRUB bootloader'
```

Comment:
Maybe something to distinguish it from the `grub2` plugin a bit more when looking at the `--list-plugins` output?

Reply:
oops, total typo from our side; will fix in next iteration